File: Tarea_8/src/Libro.java
import java.util.*;
public class Libro {
    private int id;
    private String titulo;
    private String autor;
    boolean rentado;

    public Libro(String titulo, String autor) {
        this.id = new Random().nextInt(9999) + 1; // Genera un número aleatorio entre 1 y 9999
        this.titulo = titulo;
        this.autor = autor;
        this.rentado = false;

    }
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public java.lang.String getTitulo() {
        return titulo;
    }

    public void setTitulo(java.lang.String titulo) {
        this.titulo = titulo;
    }

    public java.lang.String getAutor() {
        return autor;
    }

    public void setAutor(java.lang.String autor) {
        this.autor = autor;
    }

    public boolean isRentado() {
        return rentado;
    }

    public void setRentado(boolean rentado) {
        this.rentado = rentado;
    }
}

File: Tarea_8_PY/Libro.py
import random

class Libro:
    def __init__(self, titulo, autor):
        self.id = random.randint(1, 9999)  # Generate random ID between 1 and 9999
        self.titulo = titulo
        self.autor = autor
        self.rentado = False

    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

    def get_titulo(self):
        return self.titulo

    def set_titulo(self, titulo):
        self.titulo = titulo

    def get_autor(self):
        return self.autor

    def set_autor(self, autor):
        self.autor = autor

    def is_rentado(self):
        return self.rentado

    def set_rentado(self, rentado):
        self.rentado = rentado

    def __str__(self):
        return f"Libro: ID: {self.id}, Título: {self.titulo}, Autor: {self.autor}, Rentado: {self.rentado}"